crates/database/src/auction_orders.rs

```diff
@@ -0,0 +1,68 @@
+use {
+    crate::{auction::AuctionId, OrderUid, PgTransaction},
+    sqlx::PgConnection,
+    std::ops::DerefMut,
+};
+
+/// Included order for a given auction.
+#[derive(Debug, Clone, PartialEq, sqlx::FromRow)]
+pub struct AuctionOrder {
+    pub auction_id: AuctionId,
+    pub order_uid: OrderUid,
+}
+
+pub async fn insert(
+    ex: &mut PgTransaction<'_>,
+    orders: &[AuctionOrder],
+) -> Result<(), sqlx::Error> {
+    const QUERY: &str = r#"INSERT INTO auction_orders (auction_id, order_uid) VALUES ($1, $2);"#;
+    for price in orders {
+        sqlx::query(QUERY)
+            .bind(price.auction_id)
+            .bind(price.order_uid)
+            .execute(ex.deref_mut())
+            .await?;
+    }
```

Comment:
Why not insert all the values using a single query using a list of `VALUES`?

Reply:
updated the db schema